### PR TITLE
Updated Terms of Service

### PR DESCRIPTION
**Pull request by NC-007**

### CODE REVIEW

File: FHICORC/Locales/en.json [hunk 1]
```diff
@@ -94,6 +94,9 @@
   "HELP_ABOUT_APP": "About Kontroll av koronasertifikat",
   "HELP_ABOUT_APP_CONTENT": "The application has functionality to check that the COVID-19 certificate is valid by scanning the QR code in the COVID-19 certificate.",
   "HELP_URL": "https://www.helsenorge.no/en/covid-19-certificate/",
+  "HELP_TERMS_OF_SERVICE": "Brukervilkår",
+  "HELP_TERMS_OF_SERVICE_CONTENT": "Ved å bruke appen aksepterer du de til enhver tid gjeldene brukervilkår",
+  "HELP_TERMS_OF_SERVICE_BUTTON": "Åpne brukervilkår",
```
Comment (NC-002) [inline FHICORC/Locales/en.json:99]: These should be in english

File: FHICORC/Locales/nn.json [hunk 2]
```diff
@@ -94,6 +94,9 @@
   "HELP_ABOUT_APP": "Om Kontroll av koronasertifikat",
   "HELP_ABOUT_APP_CONTENT": "Applikasjonen inneheld funksjonalitet for å kontrollera at koronasertifikat er gyldige ved å skanna QR-koden i koronasertifikatet.",
   "HELP_URL": "https://www.helsenorge.no/koronasertifikat/",
+  "HELP_TERMS_OF_SERVICE": "Brukervilkår",
+  "HELP_TERMS_OF_SERVICE_CONTENT": "Ved å bruke appen aksepterer du de til enhver tid gjeldene brukervilkår",
+  "HELP_TERMS_OF_SERVICE_BUTTON": "Åpne brukervilkår",
```
Comment (NC-002) [inline FHICORC/Locales/nn.json:99]: Review these for NN

**Review by NC-002 — CHANGES_REQUESTED** [review]
Some translations needs correction

File: FHICORC/Locales/en.json (hunk 1 above)
Comment (NC-007) [reply FHICORC/Locales/en.json]: Fixed

File: FHICORC/Locales/nn.json (hunk 2 above)
Comment (NC-007) [reply FHICORC/Locales/nn.json]: Fixed

**Review by NC-002 — APPROVED** [review]

**Pull request merged**